docs/whitepaper.tex

```diff
@@ -247,38 +247,98 @@ \section{Tests}
 
 \todo{Should we write Apropos-Tx tests?}
 
-\subsection{Unit tests}
+\subsection{Components}
+
+Tests are broken down by the onchain component they test.
+
+\subsubsection{Config utxo tests}
+
+NFT tests
 
-The following intended behaviors need to work:
 \begin{itemize}
-  \item Initializing a protocol and pool works
-  \item queries accurately reflect how the pool was setup
-  \item valid swaps work
-  \item the queries correctly report which swaps would be valid
-  \item queries accurately reflect the change from a swap
-  \item valid liquidity changes work
-  \item queries reflect the liquidity changes
-  \item admin can kill a pool
-  \item liquidity can be extracted from a dead pool
+	\item Minting an NFT actually produces an NFT
```

Comment:
What is "an actual NFT", and how will you verify this requirement?

Reply:
I just mean the NFT shows up in the wallet after its minted.

Reply:
So you mean "it produces a token"? That seems part of the concept of "minting". Or do you mean "When minting a Pool ID, the Pool ID ends up in the pool"?